jest/preprocessor.js

```diff
@@ -2,15 +2,7 @@
 
 var babel = require('babel');
 var coffee = require('coffee-script');
-
-var tsPreprocessor = require('./ts-preprocessor');
-
-var defaultLibraries = [
-  require.resolve('./jest.d.ts'),
-  require.resolve('../src/isomorphic/modern/class/React.d.ts'),
```

Comment:
If we're no longer doing type checks, you can presumably delete these files too.

However, I think that we probably DO want to do TypeChecks because the idea was that we could start expanding and maintaining this `.d.ts` file for React and we'd want to test those. Better that it lives in our repo than somewhere else and people are already doing this work, so they can do it in the React repo.